dataeng/jobs/analytics/WarehouseTransforms.groovy

```diff
@@ -44,6 +45,16 @@ class WarehouseTransforms{
                             cleanAfterCheckout()
                         }
                     }
+                    git {
+                        remote {
+                            url('git@github.com:edx/jenkins-job-dsl.git')
+                            branch('$JOB_DSL_BRANCH')
+                        }
+                        extensions {
+                            pruneBranches()
+                            relativeTargetDirectory('jenkins-job-dsl')
+                        }
+                    }
```

Comment:
I don't think we need this as we're already reading rest of the scripts from the DSL when we seed/create the job.